### ISSUE TITLE
Allow custom file

### ISSUE THREAD

**Issue by abergmeier-dsfishlabs** [rosshemsley/SublimeClangFormat#34]
We have our custom file in a certain directory. Sadly it seems like I cannot set a custom path to that format file.

**Comment by rosshemsley** [comment]
Unfortunately I don't have much time to support new features right now, but I will happily accept pull requests.

**Comment by cenit** [comment]
clang-format usually reads custom file (.clang-format) if available in the folder and if not going up in the folder tree, looking in all the parent folders.
How does it work, as of now?

**Comment by chere005** [comment]
Right, it seems the only way to specify the clang-format is via sublime settings which is quite cumbersome when most users already have a .clang-format file sitting in their directory. It's a shame the only tool on package control does not support this, I've found this to be much more usable even with its quirks:

https://github.com/llvm-mirror/clang/blob/master/tools/clang-format/clang-format-sublime.py
